test/cpp/interop/xds_interop_client.cc

```diff
@@ -47,7 +48,7 @@ ABSL_FLAG(int32_t, num_channels, 1, "Number of channels.");
 ABSL_FLAG(bool, print_response, false, "Write RPC response to stdout.");
 ABSL_FLAG(int32_t, qps, 1, "Qps per channel.");
 // TODO(Capstan): Consider using absl::Duration
-ABSL_FLAG(int32_t, rpc_timeout_sec, 30, "Per RPC timeout seconds.");
+ABSL_FLAG(int32_t, rpc_timeout_sec, INT_MAX, "Per RPC timeout seconds.");
```

Comment:
It's not a good idea to set INT_MAX as the default deadline. It makes infrastructure failures slow/hard to be detected, especially in warmup. We may want to make it dynamically configurable through the `Configure()` RPC. I believe the timeout test would need it as well.

Reply:
as per our discussion, let's put it in the config proto, separate PR